### ISSUE TITLE
Images not displayed when the name contains spaces

### ISSUE THREAD

**Issue by gabm** [gollum/gollum#1169]
When the image filename contains spaces, the image is not rendered in the text of the wiki page but instead a link is displayed. When I click on the link, the image is displayed correctly.

## Example
1. `FirstImage.jpg`
2. `Second Image With Spaces.jpg`

## Wikipage:

```
# First Image
[[FirstImage.jpg]]

# Second Image
[[Second Image With Spaces.jpg]]
```

## Results in
First Image is displayed correctly while the second is a red link with the text `/uploads/Second Image With Spaces.jpg`

**Comment by actualjellyfish** [comment]
Does it work if you replace the spaces with dashes?

**Comment by bartkamphorst** [comment]
We're looking into changing the whole naming scheme for gollum version 5.x. Might be a while though.

**Comment by dometto** [comment]
Fixed in `v5.0`, upcoming release (might be a while).
